src/main/java/seedu/duke/commands/HelpCommand.java

```diff
@@ -10,8 +10,8 @@
 public class HelpCommand extends Command {
     public static final String COMMAND_WORD = "help";
 
-    private static final String SHOW_ALL_COMMANDS = "List of commands: 'add', 'list', 'remove', 'find',"
-            + " 'update', 'exit'."
+    private static final String SHOW_ALL_COMMANDS = "List of commands: 'add', 'list', 'remove', 'find', "
+            + "'update', 'clear', 'exit'."

```

Comment:
thank you for updating clear feature!! Now that all command features are included in the help command.